tests/common/estimates.py

```
@@ -0,0 +1,216 @@
+    return individual_effects.mean()
+
+
+def compare_treatment_effect_estimates_with_ground_truth_across_patient_numbers(
+    patient_numbers,
+    models,
+    estimators,
+    n_bootstraps=100,
+    effect_type="ATE",
+    ps_model=LogisticRegression,
+    outcome_model=LogisticRegression,
+    ps_model_kwargs={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)